soc/arc/snps_nsim/CMakeLists.txt

```diff
@@ -7,6 +10,34 @@ zephyr_compile_options_ifdef(CONFIG_FPU -mfpu=fpud_all)
 else()
 zephyr_compile_options_ifdef(CONFIG_FPU -mfpu=fpuda_all)
 endif()
+else()
```

Comment:
I know you just follow existing style, but I think indentation should be fixed.
But it is accepted if done at later time, that is after this PR.

```
if()
...
...
if()
..
else()
..
endif()
else()
...
...
```
is hard to read.